packages/react/src/components/Tabs/Tabs.js

```diff
@@ -252,14 +252,21 @@ export default class Tabs extends React.Component {
     });
 
     const tabContentWithProps = React.Children.map(tabsWithProps, tab => {
-      const { children, selected, renderContent: TabContent } = tab.props;
+      const {
+        id: tabId,
+        children,
+        selected,
+        renderContent: TabContent,
+      } = tab.props;
 
       return (
         <TabContent
+          id={!tabId ? undefined : `${tabId}__panel`}
```

Comment:
would this work as well?

```suggestion
          id={tabId && `${tabId}__panel`}
```